### PR TITLE
feature - design list view

### PR DESCRIPTION
**Pull request by at-dungduong**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/ViewModel/HomeViewModel.swift [hunk 1]
```diff
@@ -11,12 +11,28 @@ import MVVM
 import RealmSwift
 
 final class HomeViewModel: MVVM.ViewModel {
+    enum CategoryType: Int {
+        case restaurant = 0
+        case diner
+        case icecream
+
```
Comment (at-daiho) [inline MyApp/ViewModel/HomeViewModel.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

**Comment by at-daiho** [comment]
Total linter violations in pull request: 2
Serious: 0
Warnings: 2

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: Demo/Login/Login/Tabbar/HomeViewController.swift [hunk 2]
```diff
@@ -163,6 +186,10 @@ extension HomeViewController: UICollectionViewDataSource {
     }
 }
 
+extension UITextField {
```
Comment (at-tuanmai2) [inline Demo/Login/Login/Tabbar/HomeViewController.swift]: ko có gì thì xoá đi

**Pull request merged**